record_indexer/settings.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+LOCAL_RUN = os.environ.get('INDEXER_LOCAL_RUN', False)
```

Comment:
what's a local run? what's not a local run?

Reply:
Removed in commit 911551ccdd57e87851037cddd4e7fce74ba0523c - we can add it back if it becomes useful.

Reply:
Right, of course, we don't need to set `LOCAL_RUN` here because the lambda doesn't call itself as is.